pghoard/pghoard.py

```diff
@@ -297,6 +298,36 @@ def delete_remote_basebackup(self, site, basebackup, metadata):
                             chunk["chunk_filename"],
                         )
                     )
+        elif metadata.get("format") == BaseBackupFormat.delta_v1:
+            keep_hexdigests = set()
+            for delta_backup in basebackups:
+                if delta_backup["name"] == basebackup:
+                    continue
+                delta_backup_key = os.path.join(
+                    self.config["backup_sites"][site]["prefix"], "basebackup", delta_backup["name"]
```

Comment:
`self.config["backup_sites"][site]["prefix"]` is used several times here, can be extracted into a variable.